res/icons/delivery-truck.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M5 18H4a1 1 0 01-1-1v-2h12V6h4.667c1.117 0 1.6.576 1.936 1.107.594.94 1.536 2.432 2.109 3.378.188.312.288.67.288 1.035V16c0 1.121-.728 2-2 2h-1a3.001 3.001 0 01-6 0h-4a3.001 3.001 0 01-6 0zm3-1.2a1.201 1.201 0 010 2.4 1.201 1.201 0 010-2.4zm10 0a1.201 1.201 0 010 2.4 1.201 1.201 0 010-2.4zM8 8H0V6h8V5H3V4a1 1 0 011-1h9a1 1 0 011 1v10H3v-3H1V9h7V8zm9 0v3h4.715l-1.427-2.496A.999.999 0 0019.42 8H17z"/></svg>
```

Comment:
This SVG has not defined a `viewBox`. Maybe that is the source of the error?

Reply:
Unfortunately not. It happens when it is linted. See https://github.com/datenanfragen/website/issues/92#issuecomment-543612612

This will be an issue for all new svg added. I'll try and change the configuration for lint-staged but I haven't had anything to do with that before.

Reply:
It's not just that svg I tried a few more and they don't work either. The paper plane works OK after the linting.

Reply:
OK, so I will try and fix the problem in `imagemin-lint-staged`, but until then I guess we should just disable `imagemin-lint-staged` for `.svg` altogether. So if you remove that in the `package.json`, `lint-staged` should leave the delivery truck be and the build should work. I will reimplement it as soon as I fixed the problem.